test/integration/lib/update-lock.spec.ts

```diff
@@ -281,4 +284,117 @@ describe('lib/update-lock', () => {
 			);
 		});
 	});
+
+	describe('composition step actions', () => {
+		const lockdir = pathOnRoot(updateLock.BASE_LOCK_DIR);
+		const serviceLockPaths = {
+			1: [
+				path.join(lockdir, '1', 'server', 'updates.lock'),
```

Comment:
I know that `path.join` is technically correct, but I feel that for a testing context `\`${lockdir}/1/server/updates.lock\`` is a lot more readable. Is just a nitpick, keep it this way if you prefer it

Reply:
Path names simplified [here](https://github.com/balena-os/balena-supervisor/pull/2234/commits/7b9be98a7831ff63764d7a20261881535a0b1d30#diff-e7a2b32a616114c913967bbeb93301d1194c25d37b5c6d6393146a862d33b61fR354-R365)